src/styles/main.scss

```diff
@@ -1,3 +1,43 @@
 body {
   margin: 0;
+  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  text-align: center;
+  line-height: 300px;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  max-width: 1199px;
+}
+
+.box {
+  height: 300px;
+
+  &--1 {
+    background-color: #f00;
+  }
+
+  &--2 {
+    background-color: rgb(80%, 0%, 0%);
+  }
+
+  &--3 {
+    background-color: rgb(60%, 0%, 0%);
+  }
```

Comment:
Use one variable for all colors and create cycle for setting background-color to all elements